tests/unit/manage/test_views.py

```diff
@@ -1465,9 +1465,11 @@ def test_default_response(self, monkeypatch):
         monkeypatch.setattr(views, "DeleteMacaroonForm", delete_macaroon_cls)
 
         project_names = [pretend.stub()]
```

Comment:
I think this variable + associated monkeypatch (and all duplicates below) can be removed.